package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "r7insight_node",
-  "version": "3.2.0",
+  "version": "3.2.1",
```

Comment:
Had me confused here :D Version number of r7insight_node and winston lined up

Reply:
btw you probably don't want to increase the version number here and in this way, as the git tags will get out of sync. 
Also, we're quite possibly breaking compatibility here, so should this be a major?

Reply:
This change doesn't break compatibility(if you want to double check yourself, you can install this branch locally using `yarn install git+https://github.com/rapid7/r7insight_node.git#fix-module-imports --save` and see how it works with and without winston). I'll revert the package version change